alpha-build-machinery/make/examples/multiple-binaries/Makefile.test

```diff
@@ -51,3 +51,7 @@ test-rpm:
 	[[ ! -d ./_output/ ]]
 	$(MAKE) clean
 .PHONY: test-rpm
+
+test-update-codegen:
```

Comment:
you also need to register the test on line 3 as the others are, this isn't run in CI